fuse_optimizers/test/test_fixed_lag_ignition.cpp

```diff
@@ -113,11 +113,16 @@ TEST(FixedLagIgnition, SetInitialState)
 
   // Wait for the optimizer to process all queued transactions
   rclcpp::Time result_timeout = node->now() + rclcpp::Duration::from_seconds(3.0);
-  auto odom_msg = nav_msgs::Odometry::ConstPtr();
+  auto odom_msg = nav_msgs::msg::Odometry::ConstSharedPtr();
   while ((!odom_msg || odom_msg->header.stamp != rclcpp::Time(3, 0)) &&
          (node->now() < result_timeout))
   {
-    odom_msg = ros::topic::waitForMessage<nav_msgs::Odometry>("/odom", rclcpp::Duration::from_seconds(1.0));
+    // TODO(CH3): Oh no, there's no equivalent rclcpp method I think...
+    //            Probably just wait a bit?? It's in a test anyway
```

Comment:
Seems that there actually is a [wait_for_message](https://github.com/ros2/rclcpp/blob/rolling/rclcpp/include/rclcpp/wait_for_message.hpp) method in rclcpp. I'll use that when it's time to get around to fuse_optimizers. And if that fails I'll go the condition variable route (instead of futures) :grimacing: